kythe/go/extractors/golang/packages.go

```diff
@@ -150,7 +150,7 @@ func (e *Extractor) listPackages(query ...string) ([]*jsonPackage, error) {
 	// TODO(schroederc): support GOPACKAGESDRIVER
 	args := append([]string{"list",
 		"-compiler=" + e.BuildContext.Compiler,
-		"-tags=" + strings.Join(e.BuildContext.BuildTags, ","),
+		"-tags=" + strings.Join(e.BuildContext.BuildTags, " "),
```

Comment:
How does this work? Why isn't each tag after the first picked up as a new arg/flag?

Reply:
Because these are passed to `exec.Command`, I think it essentially gets quoted like "-tags=tag1 tag2" before being passed to to `go` cmd.

I agree though that spaces vs commas is weird...

Reply:
If you just did something like `go build -tags tag1 tag2` on the cmdline, it would behave like you said and see "tag2" as a new flag.